cmd/dev/newsletter/stub/changelog.html.expected

```diff
@@ -8,17 +8,17 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```

Comment:
Revert the white space things or tests will fail

Reply:
yea i am working on this, please excuse me